content/blog/pulumi-for-aws-automate-secure-manage/index.md

```diff
@@ -0,0 +1,87 @@
+
+[Read more about the Pulumi AWS Cloud Control Provider](/blog/pulumi-aws-cloudcontrol-provider/)
+
+### AWS CDK for Pulumi
+
+In 2022, Pulumi brought the ability to use AWS Cloud Development Kit (CDK) constructs from within a Pulumi deployment. For users already using AWS CDK, this provides Pulumi as a new option for orchestrating deployments in place of CloudFormation, offering improved deployment speed, integration with the full set of features of the Pulumi Platform. Even better, you can also now combine AWS CDK and Pulumi resources in a single Pulumi infrastructure as code project - passing outputs from Pulumi resources into AWS CDK constructs, and outputs from AWS CDK constructs into Pulumi resources.
+
+AWS Cloud Development Kit (CDK) on Pulumi is now generally available. This powerful integration includes:
+
+* Full support for CDK features like assets, custom resources, aspects, and context values/methods.
+* Coverage for 19 of the top 20 construct libraries, thoroughly validated through testing.
+* Support for Docker assets, so you can build/push Docker images to ECR, streamlining  the delivery of containerized applications.
+
+[Read more about AWS CDK on Pulumi](/blog/aws-cdk-on-pulumi-1.0/)
+
+## Beyond Pulumi IaC
+
+Over the last few years, our customers have shared a clear message:  IaC alone is insufficient. The complexities of modern cloud environments demand more than just IaC to manage cloud resources, improve security, and ensure compliance. In response, Pulumi's offerings have expanded into a comprehensive suite of tools that go beyond IaC.
+
+![Pulumi platform diagram](platform.png)
+
+### Pulumi Insights: Intelligent Cloud Management
+
+[Pulumi Insights](/product/pulumi-insights/) empowers organizations to tackle cloud complexity by delivering visibility, actionable insights, and AI-assisted optimization for their infrastructure. With Pulumi Insights, companies can scan and sync their entire AWS resource inventory – including resources not managed with Pulumi IaC – ensuring a complete picture of their cloud assets. Compliance checks can be run against standards such as PCI DSS, ISO 27001, and CIS benchmarks, helping maintain regulatory compliance and mitigate risks. The AI-powered [Pulumi Copilot](/product/copilot/) provides an interactive assistant to gain visibility into a team's activity, discover cost saving opportunities, get compliant, and debug cloud failures. Pulumi Insights enables companies to discover, understand, manage, and improve their cloud infrastructure on their journey towards intelligent infrastructure management.
```

Comment:
@thtmnisamnstr should we add in more market context here and related it back to the complexity we introduce at the very beginning? 

> This complexity creates significant challenges in making sense of the millions of cloud resources and resource updates per month, across hundreds of clouds, regions, and accounts. Identifying cost saving opportunities is difficult amidst soaring cloud expenditures, and maintaining compliance and security is mission critical because errors can cause catastrophic damage.

Reply:
I think it helps.